sendgrid/helpers/mail/mail.py

```diff
@@ -725,9 +730,16 @@ def add_content(self, content, mime_type=None):
         """
         if isinstance(content, str):
             content = Content(mime_type, content)
-        # Content of mime type text/plain must always come first
+        # Content of mime type text/plain must always come first, followed by text/x-amp-html and then text/html
```

Comment:
We should add a test to ensure this is true.

Also, I think `text/x-amp-html` should be last.

Reply:
Hmm, not really, if the `text/x-amp-html` is at the last, it fails in apple mail. Have tested the ordering earlier.